File: benchmarks/python/13_hashmap_build_query.py
def hashmap_build_query(n):
    m = {}
    for i in range(n):
        m[i] = i * 2.5
    s = 0.0
    for i in range(n):
        v = m.get(i)
        if v is not None:
            s += v
    return s
print(hashmap_build_query(100000))

File: benchmarks/node/13_hashmap_build_query.mjs
function hashmap_build_query(n) {
    const m = new Map();
    for (let i = 0; i < n; i++) {
        m.set(i, i * 2.5);
    }
    let sum = 0;
    for (let i = 0; i < n; i++) {
        const v = m.get(i);
        if (v !== undefined) {
            sum += v;
        }
    }
    return sum;
}
console.log(hashmap_build_query(100000));
